### PR TITLE
Update portals.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed link from intune to endpoint manager admin center

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2525

File: microsoft-365/security/mtp/portals.md [hunk 1]
```diff
@@ -48,4 +48,4 @@ While these portals are not specifically for managing security, they support var
...
 | Microsoft 365 admin center | Configure Microsoft 365 services; manage roles, licenses, and track updates to your Microsoft 365 services | [admin.microsoft.com](https://admin.microsoft.com/) |
 | Microsoft Endpoint Manager admin center | Use [Microsoft Endpoint Manager](https://docs.microsoft.com/mem/configmgr/) to manage and secure devices using combined Intune and Configuration Manager capabilities | [devicemanagement.microsoft.com](https://devicemanagement.microsoft.com/) |
-| Microsoft Intune portal | Use [Microsoft Intune](https://docs.microsoft.com/intune/fundamentals/what-is-intune) to deploy device policies and monitor devices for compliance | [portal.azure.com/#blade/Microsoft_Intune_DeviceSettings](https://portal.azure.com/#blade/Microsoft_Intune_DeviceSettings/ExtensionLandingBlade/overview)
\ No newline at end of file
+| Microsoft Intune portal | Use [Microsoft Intune](https://docs.microsoft.com/intune/fundamentals/what-is-intune) to deploy device policies and monitor devices for compliance | [endpoint.microsoft.com/#blade/Microsoft_Intune_DeviceSettings](https://endpoint.microsoft.com/#blade/Microsoft_Intune_DeviceSettings/DevicesMenu/overview)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/portals.md]:
```suggestion
| Microsoft Intune portal | Use [Microsoft Intune](https://docs.microsoft.com/intune/fundamentals/what-is-intune) to deploy device policies and monitor devices for compliance | [endpoint.microsoft.com](https://endpoint.microsoft.com/#blade/Microsoft_Intune_DeviceSettings/DevicesMenu/overview)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lomayor Please merge this PR. Thanks

**Pull request merged**